src/commandHandlers/tips/opensource.service.ts

```diff
@@ -0,0 +1,18 @@
+// TODO: read the data for tips and resources from the DB. The idea is for this data to not be hardcoded, and can be
+// modified throughout time, without needing to re-deploy the bot!
```

Comment:
Good idea 👍 . Can be hard coded for now, I like it is in a service we can easily update in a single place when we move to the DB